.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python: 2.7
```

Comment:
since you use sudo you should say ``sudo: true`` to tell travis to not run on their container based infrastructure, where sudo wont work. so said you wont get enhanced caching features of travis w/o their containers.